hq/test/logic/RetryTest.scala

```diff
@@ -12,6 +14,7 @@ class RetryTest extends WordSpec with Matchers with AttemptValues {
     val Started = "STARTED"
     val maxAttempt = 10
     val predicateF = { status: String => status == Complete }
+    val testDelay = 100.millisecond
```

Comment:
I'd divide this (and corresponding references) by 10 for the tests to keep them quick.

Reply:
👍